src/state/stake/hooks-sushi.ts

```diff
@@ -163,3 +163,42 @@ export function useInfiniteScroll(items: any[]): [number, Dispatch<number>] {
   useEffect(() => setItemsDisplayed(10), [items.length])
   return [itemsDisplayed, setItemsDisplayed]
 }
+
+export function getMultipleContracts(
+  addressList: string[],
+  ABI: any,
+  library: Web3Provider,
+  account?: string
+): Contract[] {
+  const invalidAddress = addressList.find(address => !isAddress(address) || address === AddressZero)
```

Comment:
Only need to find one invalid one

```suggestion
  const invalidAddress = addressList.some(address => !isAddress(address) || address === AddressZero)
```